sql/hive/src/test/scala/org/apache/spark/sql/hive/InsertSuite.scala

```diff
@@ -824,4 +824,28 @@ class InsertSuite extends QueryTest with TestHiveSingleton with BeforeAndAfter
       }
     }
   }
+
+  test("SPARK-28050: DataFrameWriter support insertInto a specific table partition") {
+    withTable("mc_test_pt_table") {
+      import spark.implicits._
+      spark.sql(
+        s"CREATE TABLE mc_test_pt_table (name STRING, num BIGINT) PARTITIONED BY (pt1 STRING, pt2 STRING)")
+      val result =
+        """+----+----+
+          || pt1| pt2|
+          |+----+----+
+          ||0101|0202|
+          |+----+----+
+          |only showing top 1 row
+          |""".stripMargin
+      val partionDf = spark.sparkContext
+        .parallelize(0 to 9, 2)
+        .map(f => {
+          (s"name-$f", f)
+        })
+        .toDF("name", "num")
+      partionDf.write.insertInto("mc_test_pt_table", "pt1='0101',pt2='0202'")
```

Comment:
plz fix the style: `"pt1='0101', pt2='0202'"`

Reply:
you can run `./dev/sbt-checkstyle` before opening a pr.

Reply:
"pt1='0101', pt2='0202'" is one param ,
```
$ ./dev/sbt-checkstyle
Checkstyle checks passed.
```

Reply:
oh, I see. Why not use variable-length params? e.g., `insertInto(tableName: String, partionInfo: String*)`

Reply:
Thank you for your reminder